package.json

```diff
@@ -10,8 +10,8 @@
   "scripts": {
     "dev": "webpack server",
     "build": "webpack",
-    "test": "npx jest",
-    "coverage": "jest --coverage --silent",
+    "test": "npx jest -u",
```

Comment:
This command should not change as the purpose of the tests is to check if other files were affected. If you just run the command with `-u`, then it serves no purpose.